cmd/uiVeri5ExecuteTests.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+func uiVeri5ExecuteTests(config uiVeri5ExecuteTestsOptions, telemetryData *telemetry.CustomData) {
+	// for command execution use Command
+	c := command.Command{}
+	// reroute command output to logging framework
+	c.Stdout(log.Writer())
+	c.Stderr(log.Writer())
+
+	// error situations should stop execution through log.Entry().Fatal() call which leads to an os.Exit(1) in the end
+	err := runUIVeri5(&config, &c)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func runUIVeri5(config *uiVeri5ExecuteTestsOptions, command command.ExecRunner) error {
+	envs := []string{"NPM_CONFIG_PREFIX=/home/node/.npm-global/bin"}
```

Comment:
This depends on the runtime environment. Any idea how to adapt this to the respective environment?
ˋhome/nodeˋ works well for default node container.

Reply:
I would tend to keep this as it was with extending the `PATH` by `~/npm-global/bin`.